src/contracts/core/StrategyManager.sol

```diff
@@ -90,27 +90,23 @@ contract StrategyManager is
     /// @notice Emitted when the `withdrawalDelayBlocks` variable is modified from `previousValue` to `newValue`.
     event WithdrawalDelayBlocksSet(uint256 previousValue, uint256 newValue);
 
-    modifier onlyNotFrozen(address staker) {
+    function _onlyNotFrozen(address staker) internal {
         require(
             !slasher.isFrozen(staker),
             "StrategyManager.onlyNotFrozen: staker has been frozen and may be subject to slashing"
         );
-        _;
     }
 
-    modifier onlyFrozen(address staker) {
+    function _onlyFrozen(address staker) internal {
         require(slasher.isFrozen(staker), "StrategyManager.onlyFrozen: staker has not been frozen");
-        _;
     }
 
-    modifier onlyEigenPodManager {
+    function _onlyEigenPodManager() internal {
         require(address(eigenPodManager) == msg.sender, "StrategyManager.onlyEigenPodManager: not the eigenPodManager");
-        _;
     }
 
-    modifier onlyStrategyWhitelister {
+    function _onlyStrategyWhitelister() internal {
         require(msg.sender == strategyWhitelister, "StrategyManager.onlyStrategyWhitelister: not the strategyWhitelister");
-        _;
```

Comment:
@gpsanant we previously discussed reverting these changes as part of this merge. you might want to pull from master, resolve merge conflicts and make sure this gets properly handled